app/src/main/java/com/conupods/OutdoorMaps/View/MapsActivity.java

```diff
@@ -9,76 +11,56 @@
 import com.conupods.R;
 import com.google.android.gms.common.api.ResolvableApiException;
 import com.google.android.gms.location.FusedLocationProviderClient;
 import com.google.android.gms.location.LocationCallback;
 import com.google.android.gms.location.LocationRequest;
-import com.google.android.gms.location.LocationResult;
 import com.google.android.gms.location.LocationServices;
 import com.google.android.gms.location.LocationSettingsRequest;
 import com.google.android.gms.location.LocationSettingsResponse;
 import com.google.android.gms.location.SettingsClient;
-import com.google.android.gms.maps.CameraUpdateFactory;
 import com.google.android.gms.maps.GoogleMap;
 import com.google.android.gms.maps.OnMapReadyCallback;
 import com.google.android.gms.maps.SupportMapFragment;
-import com.google.android.gms.maps.model.LatLng;
-import com.google.android.gms.maps.model.MarkerOptions;
-import com.google.android.gms.tasks.OnCompleteListener;
 import com.google.android.gms.tasks.OnFailureListener;
 import com.google.android.gms.tasks.OnSuccessListener;
 import com.google.android.gms.tasks.Task;
 
-import static android.view.inputmethod.EditorInfo.IME_ACTION_DONE;
-import static android.view.inputmethod.EditorInfo.IME_ACTION_SEARCH;
-
 public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
 
     private static final String TAG = "MapsActivity";
-    private static final float DEFAULT_ZOOM = 15f;
 
 
+    public static final int LOCATION_PERMISSION_REQUEST_CODE = 1234;
+    public static final int RESOLVABLE_API_ERROR_REQUEST_CODE = 51;
+
     private GoogleMap mMap;
-    private BuildingOverlays buildingOverlays;
+    private BuildingOverlays mBuildingOverlays;
+    private CameraController mCameraController;
 
 
     private final String COURSE_LOCATION_PERMISSION = Manifest.permission.ACCESS_COARSE_LOCATION;
     private final String FINE_LOCATION_PERMISSION = Manifest.permission.ACCESS_FINE_LOCATION;
 
     //Activity Components
     //private  Button locationBtn;
-    private View mapView;
-    private EditText searchBar;
+    private View mMapView;
+    private EditText mSearchBar;
 
 
     //Variables for logic
-    boolean permissionsGranted = false;
-    private static final int LOCATION_PERMISSION_REQUEST_CODE = 1234;
-    private static final int RESOLVABLE_API_ERROR_REQUEST_CODE = 51;
-    Location lastKnownLocation;
-    LocationCallback locationCallback;
-
-    // These could to be moved outside of the file
-    public final double SGW_LAT = 45.496080;
-    public final double SGW_LNG = -73.577957;
-    public final double LOY_LAT = 45.458333;
-    public final double LOY_LNG = -73.640450;
-
-    // LatLng objects for the campuses
-    public final LatLng SGW_CAMPUS_LOC = new LatLng(SGW_LAT, SGW_LNG);
-    public final LatLng LOY_CAMPUS_LOC = new LatLng(LOY_LAT, LOY_LNG);
+    private boolean mPermissionsGranted = false;
+    private Location lastKnownLocation;
+    private LocationCallback locationCallback;
```

Comment:
member variables

Reply:
Thanks for noticing, these should have been deleted